src/services/ebic/index.jsx

```
@@ -0,0 +1,306 @@
+          description: Yup.string().required("Ebic is required"),
+          sub_category: Yup.string().required("Sub Category is required"),
+          image: Yup.string().required("Image is required"),
+        })}
+        onSubmit={handleAddEbic}
+      >
+        {({ handleSubmit, values, handleChange, setFieldValue }) => (
+          <div className="flex-col-start gap-5">
+            <div
+              className={`relative capture-container ${borderColor}`}
+              onClick={() => setIsEmojiModalOpen(true)}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)